api/query/cache/service/main.go

```diff
@@ -200,6 +200,14 @@ func searchHandler(w http.ResponseWriter, r *http.Request) {
 	if len(missing) != 0 {
 		resp.IgnoredRuns = missing
 	}
+
+	if _, showMetadata := urlQuery[shared.MetadataFlag]; showMetadata {
+		var netClient = &http.Client{
```

Comment:
We have a method in `shared` for getting an http client (on the AppEngineAPI).

Reply:
This is part of the searchcache that runs on AppEngine Flex, so no urlfetch.